app/src/main/java/org/mozilla/focus/web/ClassicWebViewProvider.java

```diff
@@ -79,7 +82,7 @@ private static void setupView(WebView webView) {
     }
 
     @SuppressLint("SetJavaScriptEnabled") // We explicitly want to enable JavaScript
-    private static void configureDefaultSettings(Context context, WebSettings settings) {
+    private void configureDefaultSettings(Context context, WebSettings settings) {
```

Comment:
\[PMD\] INFO: The method 'configureDefaultSettings\(Context, WebSettings\)' has a NCSS line count of 19\.